Makefile

```diff
@@ -163,39 +174,151 @@ build-tensorflow-ngc:
 		-t $(DOCKERHUB_REGISTRY)/$(NGC_TF_HPC_REPO):$(SHORT_GIT_HASH) \
 		.
 
+
+ROCM_DEEPSPEED_VERSION==0.14.4
+WITH_MPICH=1
+export ROCM61_TORCH_TF_ENVIRONMENT_NAME_DEEPSPEED := $(ROCM_61_PREFIX)pytorch-2.0-tf-2.10-rocm-deepspeed
+.PHONY: build-pytorch-infinityhub
+build-pytorch-infinityhub:
+	docker build --shm-size='1gb' -f Dockerfile-infinityhub-pytorch \
+                --build-arg BASE_IMAGE="rocm/pytorch:rocm6.1_ubuntu22.04_py3.10_pytorch_2.1.2" \
+                --build-arg DEEPSPEED_PIP="deepspeed==$(ROCM_DEEPSPEED_VERSION)" \
+                -t $(DOCKERHUB_REGISTRY)/$(INFINITYHUB_PYTORCH_REPO):$(SHORT_GIT_HASH) \
+                .
+	docker build --shm-size='1gb' -f Dockerfile-infinityhub-hpc \
+                --build-arg BASE_IMAGE=$(DOCKERHUB_REGISTRY)/$(INFINITYHUB_PYTORCH_REPO):$(SHORT_GIT_HASH) \
+                --build-arg WITH_MPICH=$(WITH_MPICH) \
+                -t $(DOCKERHUB_REGISTRY)/$(INFINITYHUB_PYTORCH_HPC_REPO):$(SHORT_GIT_HASH) \
+                .
+
+
 ifeq ($(WITH_MPICH),1)
-ROCM56_TORCH13_MPI :=pytorch-1.3-tf-2.10-rocm-mpich
+ROCM61_TORCH13_MPI :=pytorch-1.3-tf-2.10-rocm-mpich
 else
-ROCM56_TORCH13_MPI :=pytorch-1.3-tf-2.10-rocm-ompi
+ROCM61_TORCH13_MPI :=pytorch-1.3-tf-2.10-rocm-ompi
 endif
-export ROCM56_TORCH13_TF_ENVIRONMENT_NAME := $(ROCM_56_PREFIX)$(ROCM56_TORCH13_MPI)
-.PHONY: build-pytorch13-tf210-rocm56
-build-pytorch13-tf210-rocm56:
+export ROCM61_TORCH13_TF_ENVIRONMENT_NAME := $(ROCM_60_PREFIX)$(ROCM61_TORCH13_MPI)
+.PHONY: build-pytorch13-tf210-rocm60
+build-pytorch13-tf210-rocm60:
```

Comment:
I'm a bit confused. The base image is on rocm6.0 and pytorch-1.3-tf-2.10-rocm-mpich/ompi is on rocm6.1?

Reply:
Bumped these all to rocm6.1